website/project/model.py

```diff
@@ -1411,6 +1411,36 @@ def set_node_license(self, license_id, year, copyright_holders, auth, save=False
         if save:
             self.save()
 
+    def subscribe_user_to_notifications(self, user):
+        """ Update the notification settings for the creator or contributors
+
+        :param user: User to subscribe to notifications
+        """
+        from website.notifications.utils import to_subscription_key
+        from website.notifications.utils import get_global_notification_type
+        from website.notifications.model import NotificationSubscription
+        from website.notifications.constants import NODE_SUBSCRIPTIONS_AVAILABLE
```

Comment:
Trivial, but `[this line] as events` obviates the var assignment line immediately below.